addons/sale_stock/models/account_invoice.py

```diff
@@ -17,6 +17,34 @@ def _get_last_step_stock_moves(self):
             rslt += invoice.mapped('refund_invoice_id.invoice_line_ids.sale_line_ids.order_id.picking_ids.move_lines').filtered(lambda x: x.state == 'done' and x.location_id.usage == 'customer')
         return rslt
 
+    def _get_invoice_snln_values(self):
```

Comment:
_get_invoiced_lot_values would be more appropriate

Reply:
""" Get and prepare data to show a table of invoiced lot on the invoice's report. """